.github/actions/todb-tests/entrypoint.sh

```diff
@@ -18,7 +18,7 @@
 
 set -e
 
-cd traffic_ops/app/db/migrations;
+cd ../../../traffic_ops/app/db/migrations;
```

Comment:
From the GitHub Action logs:
```shell
/entrypoint.sh: line 21: cd: ../../../traffic_ops/app/db/migrations: No such file or directory
```
The current working directory of GitHub Actions is the root directory of the project, so no need to `cd` out of any directories first.

Reply:
I think your comment came just when my update must have gone through. It has been resolved.